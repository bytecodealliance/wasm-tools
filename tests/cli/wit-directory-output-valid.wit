// RUN: component wit % --out-dir %tmpdir | component wit %tmpdir

package a:b {
  interface foo {}

  world c {
    import foo;
    import a:c/foo;
  }
}

package a:c {
  interface foo {}
}
