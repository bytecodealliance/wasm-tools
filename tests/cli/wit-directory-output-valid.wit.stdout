package a:c {
  interface foo {
  }

}


package a:b {
  interface foo {
  }

  world c {
    import foo;
    import a:c/foo;
  }
}
